### DISCUSSION TITLE
High Cpu usage

### DISCUSSION THREAD

**Discussion by edouardkleinhans** [Jays-Home-Assistant-Add-ons/j-addon-influxdb2#6] [Q&A]
Hello, 
I've installed this addon on my server and i see an increasing cpu usage.

Do you have any idea how to reduce this behaviour ?

**Comment by jantoney** [comment]
Hey @edouardkleinhans, What CPU do you have x86/ARCH/??
Any other specs on the system please?

**Reply by edouardkleinhans** [reply]
X86 celeron j4505

**Reply by jantoney** [reply]
Are you pushing much data into it?
Where are you seeing this CPU increase? Under Configuration > System > Core? OR under the influx db database add-on it's self?

Have you done any troubleshooting? Uninstalled to confirm it's this?
Do you also have influx-V1 installed at the same time?

I'm really sorry, but I don't think I have any hardware that low spec'd to test with.
My HA runs on a 2015 Intel NUC with an Intel® Core™ i3-6100U Processor and I'm seeing everything average around 0.1-1.8% CPU on the CORE and the add-on it's self with everything running in the background.

What are you calling 'high CPU?'

Are you on a current version of HA?
